ansible/roles/gpu/files/Dockerfile.libnvidia

```diff
@@ -0,0 +1,42 @@
+# FIXME(tillt): Using golang as a base image seems stupid - better use Debian or Ubuntu or something.
+ARG golang:1.16
+FROM golang:1.16
+
+# FIXME(tillt): This is mostly a copy of https://gitlab.com/nvidia/container-toolkit/libnvidia-container/-/blob/master/mk/Dockerfile.debian
+# FIXME(tillt): Maybe we can find built images of the above -  overwriting the startup command (as it does a `make dist`)
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        apt-utils \
+        bmake \
+        build-essential \
+        bzip2 \
+        ca-certificates \
+        curl \
+        devscripts \
+        dh-make \
+        fakeroot \
+        git \
+        gnupg2 \
+        libcap-dev \
+        libelf-dev \
+        libseccomp-dev \
+        lintian \
+        lsb-release \
+        m4 \
+        pkg-config \
+        xz-utils && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
What is the exact purpose of this `rm`? Is this necessary to prevent automatic updates during the`libnvidia-container`  build and, if yes, then is it really sufficient? I would expect that `/etc/apt/sources*` has to be removed as well (or something similar - removing the responsible cron job, etc ), otherwise `apt update` will happily recreate these.

Reply:
> What is the exact purpose of this `rm`? 
Unsure myself - taken from the original

> Is this necessary to prevent automatic updates during the`libnvidia-container` build and, if yes, then is it really sufficient? I would expect that `/etc/apt/sources*` has to be removed as well (or something similar - removing the responsible cron job, etc ), otherwise `apt update` will happily recreate these.
Great points indeed.